sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/CreateHiveTableAsSelectCommand.scala

```diff
@@ -94,7 +94,7 @@ case class CreateHiveTableAsSelectCommand(
     Seq.empty[Row]
   }
 
-  override def argString: String = {
+  override def argString(isLeaf: Boolean): String = {
```

Comment:
What is the value of adding `isLeaf`? Don't leaf nodes already know they are? Why not just have leaf nodes call `outputStringWithTypes` or something and avoid all of the API changes?

Reply:
thanks, you are right. I won't change the current implementation, since if we need to change behavior according to a flag (for instance if we decide to add the info only if the extended mode is set) these changes are needed. If not, I will follow this comment, thanks.